docker-compose.yaml

```diff
@@ -8,12 +7,11 @@ services:
     ports:
       - "27017:27017"
       - "28017:28017"
-    expose:
-      - "27017"
-      - "28017"
 
   app:
     image: deadrobots/pixi:app
+    depends_on:
+      - db
     ports:
       - "8000:8000"
-      - "8090:8090"
+      - "8090:8090"
```

Comment:
Was this removed on purpose? It seems at least github would like it to be there:
![image](https://user-images.githubusercontent.com/7570458/39521998-df0bf838-4dde-11e8-9cac-71adac5b2a85.png)